logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -150,15 +150,18 @@ internal class ConnectionDataSource(
     override suspend fun ignoreConnectionRequest(userId: UserId): Either<CoreFailure, Unit> =
         updateRemoteConnectionStatus(userId, IGNORED)
             .flatMapLeft {
-                if (it is NetworkFailure.FederatedBackendFailure.FailedDomains)
-                    wrapStorageRequest { connectionDAO.getConnectionByUser(userId.toDao()) }
-                        .map { connectionEntity ->
-                            val updatedConnection = connectionMapper.fromDaoToModel(connectionEntity).copy(status = IGNORED)
-                            handleUserConnectionStatusPersistence(updatedConnection)
-                        }
-                else it.left()
+                if (it is NetworkFailure.FederatedBackendFailure.FailedDomains) Either.Right(Unit) else it.left()
+            }
+            .map {
```

Comment:
in this case, you can use onSuccess to update local DB when the request is successful  

```suggestion
            .onSuccess {
```

Reply:
I just did not want to do the `map { Unit }` in the end.
```
            .onSuccess {
                setConnectionStatus(userId, IGNORED)
            }.map { Unit }
```
But, agree, that is more readable.